core-bundle/src/DependencyInjection/ContaoCoreExtension.php

```diff
@@ -141,6 +141,10 @@ public function load(array $configs, ContainerBuilder $container): void
         $container->setParameter('contao.intl.countries', $config['intl']['countries']);
         $container->setParameter('contao.insert_tags.allowed_tags', $config['insert_tags']['allowed_tags']);
         $container->setParameter('contao.sanitizer.allowed_url_protocols', $config['sanitizer']['allowed_url_protocols']);
+        $container->setParameter('contao.altcha.algorithm', $config['altcha']['algorithm']);
```

Comment:
I don't think we need these parameters to be present in the container. They are not reused. Only our Altcha service needs them. We can directly configure the service and set the configuration arguments. See other examples in the `ContaoCoreExtension`. Something along the lines of

```php
        if (!$container->hasDefinition('contao.altcha.altcha')) {
            return;
        }

        $altcha = $container->getDefinition('contao.altcha.altcha');
        $altcha->setArgument(3, $config['altcha']['algorithm']);
        ....
```